src/api/RequestApi.ts

```diff
@@ -1,10 +1,55 @@
-import { spWebContext } from "../providers/SPWebContext";
 import { ApiError } from "./InternalErrors";
 import { IItemUpdateResult } from "@pnp/sp/items";
 import { EMPTYPES } from "../constants/EmpTypes";
 import { worklocation } from "../constants/WorkLocations";
 import { SPPersona } from "../components/PeoplePicker/PeoplePicker";
 
+import { spWebContext } from "../providers/SPWebContext";
+import { useQuery } from "@tanstack/react-query";
+
+export interface IRequest {
+  Id: number;
+  empName: string;
+  eta: string | null;
+  completionDate: string | null;
+}
```

Comment:
We have an IInForm that has this data --  we should just use that instead -- probably worth renaming it to IInRequest